MessageGeneration/ActionCodeGenerator.cs

```diff
@@ -165,7 +164,7 @@ public void WrapActionSections(string type)
         writer.WriteLine($"{TWO_TABS}public const string RosMessageName = \"{_rosPackageName}/{wrapperName}\";");
```

Comment:
ONE_TAB og TWO_TABS funker vel ikke dersom du fjerner dem slik du gjorde her?

Reply:
`using static RosNet.MessageGeneration.Utilities;`løser det, de var definert to steder